news/_posts/2020-01-07-critical-security-vulnerability-in-prestashop-modules.md

```diff
@@ -28,6 +28,8 @@ Connect to your shop via FTP or shell access, and look at the “vendor” direc
 
 If there’s a directory called “phpunit” inside the aforementioned directories, your shop may be vulnerable.
 
+**Warning: don't touch anything else or you might broke your shop**
+
```

Comment:
Should we had something about the Symfony folder as well?

Reply:
The folder /vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/ is safe, do not delete it.

Reply:
can you suggest something ? I don't see what you mean

Reply:
You can add this sentence to the Warning:
"The folder /vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/ is safe, do not delete it."